database/migrations/2016_03_13_110032_create_questions_table.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateQuestionsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {   
+        // A question is identified by its id, event's id and the organization's id.
+        Schema::create('questions', function (Blueprint $table)
+        {   
+            $table->increments('id');
+            $table->integer('user_id')->unsigned()->index();
+            $table->foreign('user_id')->references('id')->on('users')
+            ->onDelete('cascade');
+            $table->integer('organization_id')->unsigned()->index();
+            $table->foreign('organization_id')->references('id')->on('organizations')->onDelete('cascade');
+            $table->integer('event_id')->unsigned()->index();
+            $table->foreign('event_id')->references('id')->on('events')->onDelete('cascade');
+            $table->string('question');
+            $table->text('question_body')->nullable();
+            $table->text('answer')->nullable();
+            $table->dateTime('answered_at')->nullable();
+            $table->timestamps();
```

Comment:
I did attempt that but it gave me an error because it saw duplicate 'created at' columns, I think from my very naive perspective concerning laravel that making two timestamps may mess things up.